README.md

```diff
@@ -162,6 +162,14 @@ You can skip these and only define your own units by doing:
 gem 'measured', require: 'measured/base'
 ```
 
+### Shortcut syntax
```

Comment:
I'm not convinced of that, as it is just a convenience shortcut. I like to be explicit that you are initializing a new instance of the specific class.